robot_skills/src/robot_skills/robot_part.py

```diff
@@ -210,9 +223,24 @@ def process_hardware_status(self, diagnostic_dict):
 
     def reset(self):
         """
-        Reset body part. This should be implemented in the subclass. Should always return a bool.
+        Reset body part and all its components, this function should not be overwritten.
```

Comment:
I think it is important to take this into account because a robot will reset all of its parts, but not the subparts of its components. This is needed to keep the same behaviour